BIDebugEngine/BIDebugEngine/Debugger.h

```diff
@@ -27,7 +27,7 @@ extern MissionEventType currentEventHandler;
 
 struct BreakStateInfo {
     const DebuggerInstructionInfo* instruction{ nullptr };
-    BreakPoint* bp{ nullptr };
+    BreakPoint* bp{ nullptr };
```

Comment:
huh?

Reply:
No idea? Did white space change somehow? I don't see it